app/controller/button/SpatialQueryButton.js

```diff
@@ -80,14 +87,52 @@ Ext.define('CpsiMapview.controller.button.SpatialQueryButtonController', {
         }
     },
 
+    /**
+    * Normalizes the given degree value.
```

Comment:
IMHO the method description is misleading. Could you please provide a more precise one?

Reply:
@ahennr - thanks for spotting this! It was a copy/paste from a different function..